test/js/pageobjects/DocumentUpload.js

```diff
@@ -0,0 +1,18 @@
+import Base from './BasePage.js'
+
+class DocumentUpload extends Base{
+    getUploadInput() { return (async ()=>{
+      const input = this.$('.onfido-sdk-ui-CustomFileInput-input')
+      this.driver.executeScript(function(el) {
+        el.setAttribute('style','display: block !important')
+      },input)
+      return input
+    })()}
+
+    getUploadButton() { return this.$('.onfido-sdk-ui-Uploader-button')}
+
+    copy = (lang="en") =>
```

Comment:
Where are you using this? And how is it going to work with multiple languages?

Reply:
not used yet, I was just playing around with the design. for multiple languages it's simple, just pass the language code